### ISSUE TITLE
About Training

### ISSUE THREAD

**Issue by HaoHong7** [yosungho/LineTR#17]
Hi, I'm wondering why the loss function hasn't been changing while I'm training with the source code you provided?

**Comment by HaoHong7** [comment]
I followed the steps provided in the readme and didn't make any changes to the code, the dataset used is the Oxford and Paris dataset, I looked at the visualization of the dataset after the self-supervision and the line correspondences are correct, I don't know why the loss function hasn't been transformed during the training and the results on the validation set are poor.

**Comment by yosungho** [comment]
Hi HaoHong. I tested to train the model today, and the model was overfitting well with the training dataset. 
Not enough training data would cause poor validation scores, in general. I recommend increasing the training data so the model can have generalization ability. The loss would be moving slowly since the code is using triplet loss. Please note that even if the loss score is not moving, the distance between positive and hard-negative samples could be increased.

**Comment by HaoHong7** [comment]
Thanks for your reply, I will double check my question. If I have any further questions I may still need your help, I wonder if you would like to leave your email, I have emailed you at the email you left in the LineTR paper but it seems you don't use it anymore.Thanks again!

**Comment by HaoHong7** [comment]
Hi, I'm sorry to bother you again.
You mentioned earlier that the training dataset can also have an impact on the training results. I'd like to know what dataset you used. (What dataset was used to train the LineTR_weight.pth weight provided in the GitHub repository?) Currently, I am training with the Oxford & Paris dataset directly, but I've noticed that the weights I trained perform significantly worse when running match_line_pairs.py compared to the weights you provided.
